lib/bokken/uploaders/avatar.ex

```diff
@@ -7,13 +7,23 @@ defmodule Bokken.Uploaders.Avatar do
 
   @versions [:original, :thumb]
   @extension_whitelist ~w(.jpg .jpeg .gif .png)
+  @max_file_size 50_000_000
```

Comment:
Don't you think 50MB is too much? 😅
An avatar should have a cap of 4MB, for example.
@ruioliveira02 What's your take regarding this?